source/_includes/common-tasks/third-party-addons.md

```diff
@@ -6,6 +6,12 @@ Home Assistant allows anyone to create add-on repositories to share their add-on
 https://github.com/home-assistant/hassio-addons-example
 ```
 
+You can also add a development branch from a given repository by appending its name following a hashtag.
+
+```text
+https://github.com/home-assistant/hassio-addons-example#dev
+```
```

Comment:
I don't think that this is a general end-user use case, but a developer use case.

This does not belong in the end-user docs IMHO, more a things for developer docs or in documentation for repos that use such a structure (which is not common).

I think we should revert this, WDYT @c0ffeeca7 ?

../Frenck

Reply:
True that. Fair point.